lib/rag/ai/cohere.ex

```diff
@@ -57,11 +57,17 @@ defmodule Rag.Ai.Cohere do
   end
 
   @impl Rag.Ai.Provider
-  def generate_text(%__MODULE__{} = provider, prompt, _opts \\ []) do
+  def generate_text(%__MODULE__{} = provider, prompt, opts \\ []) do
+    stream? = Keyword.get(opts, :stream, false)
```

Comment:
This should take the stream from opts and use it to set the streaming options in the request parameters